packages/ai/src/requests/schema-builder.test.ts

```diff
@@ -81,6 +81,93 @@ describe('Schema builder', () => {
       nullable: false
     });
   });
+  describe('Schema.array', () => {
+    it('builds an array schema with basic items', () => {
+      const schema = Schema.array({
+        items: Schema.string()
+      });
+      expect(schema.toJSON()).to.eql({
+        type: 'array',
+        nullable: false,
+        items: {
+          type: 'string',
+          nullable: false
+        }
+      });
+    });
+
+    it('builds an array schema with items and minItems', () => {
+      const schema = Schema.array({
+        items: Schema.number(),
+        minItems: 1
+      });
+      expect(schema.toJSON()).to.eql({
+        type: 'array',
+        nullable: false,
+        items: {
+          type: 'number',
```

Comment:
Do you think we should use the `SchemaType` enums for these bits?

Reply:
Yes, I think we should. I can do that in a separate PR.